utils/build-presets.ini

```diff
@@ -1234,10 +1234,15 @@ verbose-build
 build-ninja
 build-swift-stdlib-unittest-extra
 
-# When producing a package, don't copy the Swift Resource/ directory.
-# This is mainly a space optimization.
+# When building for an Xcode toolchain, don't copy the Swift Resource/ directory
+# into the LLDB.framework. LLDB.framework will be installed alongside a Swift
+# compiler, so LLDB should use its resource directory directly.
+# Also, to reduce the size of the final toolchain, limit debug info to be
+# line-tables only.
 extra-cmake-options=
    -DLLDB_FRAMEWORK_COPY_SWIFT_RESOURCES=0
+   -DCMAKE_C_FLAGS="-gline-tables-only"
+   -DCMAKE_CXX_FLAGS="-gline-tables-only"
```

Comment:
This looks scary to me, but hopefully just because I don't know how to read this .ini file :-)

Can you confirm that this only affects the ci.swift.org downloadable toolchain packages, and no other toolchains?

Reply:
We still want all other toolchains to build with full debug info.

Reply:
This is under 

```#===------------------------------------------------------------------------===#
# OS X Package Builders
#===------------------------------------------------------------------------===#
[preset: mixin_osx_package_base]
```

Mishal, can you confirm it's safe?

Reply:
This is fine because it will help all preset using mixin

Reply:
Just be sure: I'm worried that this will apply to _too many_ presets, not too few.

Reply:
If we're uncertain, we should remove this. I can't picture in my mind the whole matrix of interactions with mixin et similia. It's just fundamentally odd this applies only when we don't run tests, so that's the attempt of unifying.

Reply:
Also, I'm curious, what other toolchains you're talking about?

Reply:
Adrian is talking about our internal release toolchains. @shahmishal can you confirm whether this is a noop for our internal builds?

Reply:
We don't use these preset for internal builds.

Reply:
I guess this is resolved.